docs/index.md

```diff
@@ -840,6 +844,36 @@ cssVars({
 });
 ```
 
+### onFinally
+
+- Type: `function`
+- Arguments:
+  1. **hasChanged**: A `boolean` indicating if the last ponyfill call resulted in a style change
+  1. **hasNativeSupport**: A `boolean` indicating if the current browser provides [native support](https://caniuse.com/#feat=css-variables) for CSS custom properties
+  1. **benchmark**: A `number` representing to the ponyfill execution time in milliseconds
+
+Callback after changes have been applied in modern and legacy browsers.
```

Comment:
Shouldn't this be called whether or not changes have been applied and regardless of browser? This wording makes it sound like it won't be called on every invocation of `cssVars()`

Reply:
There's a valid argument that it should. 

Here's what I was mulling over while implementing:

* When `updateDOM: false`...
  * ...and `onlyLegacy: true`...
    * ...modern browsers are ignored
    * ...legacy browsers are treated the same as `onlyLegacy: false`, and...
  * When `onlyLegacy: false`...
    * ...modern browsers are treated like legacy browsers, and...
    * ...legacy browsers receive ponyfill output via `onComplete`

Said another way, the only useful thing the ponyfill does when `updateDOM: false` is transform CSS in legacy browsers (or modern browsers treated as legacy when `onlyLegacy: false`), and the result of that process is only available to the `onComplete` callback. So, what purpose does the `onFinally` callback serve if the DOM isn't being updated? This is what lead to the current implementation of the callback only being invoked when the charges are applied to the DOM (`updateDOM: true`).

That being said, the `onFinally` callback name was meant to align with behavior of  [`try...catch...finally`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch) and [Promise.prototype.finally()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/finally), both of which are invoked regardless of what happens beforehand in either the `try...catch` statement or the promise chain. In this context, the `onFinally` callback should also fire in all scenarios.

Let me know what you think about the above. I'm happy to make the change.